lib/Module/Install/Metadata.pm

```diff
@@ -533,6 +534,7 @@ sub __extract_license {
 		'OpenSSL License'                    => 'unrestricted', 1,
 		'SSLeay License'                     => 'unrestricted', 1,
 		'zlib License'                       => 'open_source',  1,
+		'EUPL'                               => 'eupl',         1,
```

Comment:
That would have to be 'open_source', since we have a restricted list of allowed values.